pkg/mountutil/mountutil.go

```diff
@@ -56,6 +56,7 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		options  []string
 	)
 
+	s = strings.TrimLeft(s, ":")
```

Comment:
I'm confused. This code seems for parsing `-v` but the added test uses `--mount`.

Reply:
`--mount` also deal with here
https://github.com/containerd/nerdctl/blob/7dbbc311506c70edcfea9e4f300b4e1309c3ddda/pkg/mountutil/mountutil_linux.go#L418-L427